src/main/java/net/mcreator/generator/setup/folders/Pre1210FolderStructure.java

```diff
@@ -49,6 +49,7 @@ protected Pre1210FolderStructure(GeneratorFlavor flavor, Workspace workspace) {
 			case ITEM -> "/textures/item";
 			case ENTITY -> "/textures/entities";
 			case ARMOR -> "/textures/models/armor";
+			case HORSE_ARMOR -> "/textures/entity/hose/armor";

```

Comment:
For such specific textures, we usually just copy them to the right location where you just reuse e.g. entity, especially since it is single texture. See e.g. painting on how we handle them with copying to the right location and using OTHER for paitings instead of another texture category